<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xmlimportDialogBase</class>
 <widget class="QDialog" name="xmlimportDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xml-Transfer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-116</y>
        <width>493</width>
        <height>668</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string comment="Nach dem Import eine Projektdatei neu erzeugen und speichern (optional)"/>
         </property>
         <property name="title">
          <string>Schadensklassen</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <widget class="QComboBox" name="comboBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>411</width>
            <height>22</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Anzeigen in Fließrichtung rechts der Haltung</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Anzeigen in Untersuchungsrichtung rechts der Haltung</string>
           </property>
          </item>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>130</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>130</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string comment="Pfade der Import- und Exportdatenbank"/>
         </property>
         <property name="title">
          <string>Datenquelle</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <widget class="QPushButton" name="pb_import">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>40</y>
            <width>30</width>
            <height>20</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ITWH-Quelldatenbank mit Kanalnetzdaten auswählen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="tf_import">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>410</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Quelldatenbank mit Kanaldaten aus HYSTEM-EXTRAN&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>301</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>DWA-XML-Datei (*.xml):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>451</width>
            <height>43</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Tahoma</family>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Projektionssystem:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QgsProjectionSelectionWidget" name="epsg" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QDialogButtonBox" name="button_box">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Datenimport starten ...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
         <property name="centerButtons">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string comment="Pfade der Import- und Exportdatenbank"/>
         </property>
         <property name="title">
          <string>Quelle TV-Befahrungen</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <widget class="QPushButton" name="pb_import_2">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>40</y>
            <width>30</width>
            <height>20</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ITWH-Quelldatenbank mit Kanalnetzdaten auswählen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="tf_import_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>410</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Quelldatenbank mit Kanaldaten aus HYSTEM-EXTRAN&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>301</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Verzeichnis mit den Daten der TV-Befahrungen (Bilder)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="tf_import_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>410</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Quelldatenbank mit Kanaldaten aus HYSTEM-EXTRAN&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>301</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Verzeichnis mit den Daten der TV-Befahrungen (Videos)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="pb_import_3">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>90</y>
            <width>30</width>
            <height>20</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ITWH-Quelldatenbank mit Kanalnetzdaten auswählen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string comment="Pfade der Import- und Exportdatenbank"/>
         </property>
         <property name="title">
          <string>Zu importierende Daten</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>131</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Stammdaten</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>131</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Hausanschlüsse</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>131</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Zustandsdaten</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string comment="Nach dem Import eine Projektdatei neu erzeugen und speichern (optional)"/>
         </property>
         <property name="title">
          <string>Projektdatei erzeugen (optional)</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>301</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Projektdatei (*.qgs):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="pb_project">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>40</y>
            <width>30</width>
            <height>20</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pfad und Name der Projektdatei festlegen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="tf_project">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>410</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pfad der neu erzeugten Projektdatei (optional)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string comment="Pfade der Import- und Exportdatenbank"/>
         </property>
         <property name="title">
          <string>Datenbank-Verbindungen</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>23</y>
            <width>183</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Datenziel: QKan-Datenbank (*.sqlite):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="tf_database">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>411</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;QKan - Zieldatenbank (wird ggfs. neu angelegt)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="pb_database">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>50</y>
            <width>31</width>
            <height>23</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Zieldatenbank auswählen und optional erstellen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>xmlimportDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>xmlimportDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
